test/graphiteThreshold.check.spec.js

```diff
@@ -33,73 +32,101 @@ describe('Graphite Threshold Check', function(){
 		check.stop();
 	});
 
+	context('Upper threshold enforced', function () {
+
+		it('Should be healthy if below upper threshold', function (done) {
+			mockGraphite([10]);
+			check = new Check(getCheckConfig({
+				threshold: 11
+			}));
+			check.start();
+			setTimeout(() => {
+				expect(mockFetch.firstCall.args[0]).to.contain('from=-10min&target=maxSeries(metric.200)');
```

Comment:
maybe separate test for the correct url (try and keep tests to just test one thing, so you know exactly what's broken)